src/pages/Cafeteria/MobileCafeteriaPage/components/MobileMealImage/index.tsx

```diff
@@ -0,0 +1,39 @@
+import { cn } from '@bcsdlab/utils';
+import { Dining } from 'interfaces/Cafeteria';
+import { ReactComponent as NoMeals } from 'assets/svg/no-meals-mobile.svg';
+import { ReactComponent as NoPhoto } from 'assets/svg/no-photography-mobile.svg';
+import styles from './MobileMealImage.module.scss';
+
+interface Props {
+  meal: Dining,
+  handleImageClick: (item: Dining) => void,
+}
+
+export default function MobileMealImage({ meal, handleImageClick }: Props) {
+  const isABC = ['A코너', 'B코너', 'C코너'].includes(meal.place);
+  const hasImage = !!meal.image_url;
+  const isBreakfast = meal.type === 'BREAKFAST';
+  const isBoxVisible = !(!isABC || (isBreakfast && !hasImage));
```

Comment:
부정에 부정문이 들어가서 이해하기 어려울 수 있을것 같아요. 요렇게도 사용할 수 있겠네요
```suggestion
  const isBoxVisible = isABC && (!isBreakfast || hasImage);
```

Reply:
확실히 이쪽이 더 이해하기 쉽군요! ABC이면서 / 아침이 아니거나 이미지가 있으면 칸이 보임!